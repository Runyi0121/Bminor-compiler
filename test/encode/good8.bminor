"Read\0x0c the\n in\nstruc\ntions care\nfully\n"