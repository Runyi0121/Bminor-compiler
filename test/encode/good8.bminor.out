"Read\f the\n in\nstruc\ntions care\nfully\n"
